cmd/skywire-cli/commands/log/root.go

```diff
@@ -38,16 +40,22 @@ var (
 	logOnly        bool
 	surveyOnly     bool
 	deleteOnErrors bool
+	fetchFile      string
+	fetchFrom      string
+	writeDir       string
 )
 
 func init() {
 	logCmd.Flags().SortFlags = false
 	logCmd.Flags().StringVarP(&env, "env", "e", "prod", "deployment to get uptimes from")
 	logCmd.Flags().BoolVarP(&logOnly, "log", "l", false, "fetch only transport logs")
 	logCmd.Flags().BoolVarP(&surveyOnly, "survey", "v", false, "fetch only surveys")
+	logCmd.Flags().StringVarP(&fetchFile, "file", "f", "", "fetch only a specific file from all online visors")
+	logCmd.Flags().StringVarP(&fetchFrom, "pks", "k", "", "fetch only from specific public keys ; semicolon separated")
```

Comment:
the default behavior fetches the transport bandwidth logs and survey after health is successfully fetched.

Those flags simply modify the behavior to omit fetching either the survey or the bandwidth logs

I suppose those could be eliminated later if desired, however there is less typing involved with the way the code is currently to do either of those things. They aren't used regularly, it's just an effort to give more granularity to the default behavior - in case it's ever needed.